### PR TITLE
fix(visualization-server): Convert Visualization Server image into a build config

### PR DESCRIPTION
**Pull request by rimolive**
**Which issue is resolved by this Pull Request:** 
RHODS-4992

**Description of your changes:**
Convert Visualization Server image into a build config, using the pre-build tensorflow notebook images as base for building viz server.

**Environment tested:**

* Python Version (use `python --version`): 3.10
* Tekton Version (use `tkn version`): 0.21.0
* Kubernetes Version (use `kubectl version`): 1.6.0
* OS (e.g. from `/etc/os-release`): Fedora 35

**Checklist:**
- [X] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Review by anishasthana — COMMENTED** [review]
/lgtm

File: manifests/opendatahub/base/buildconfigs/vizualization-server.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    dockerStrategy:
+      from:
+        kind: ImageStreamTag
+        name: 'tensorflow:py3.8-cuda-11.4.2-2'
+      dockerfilePath: backend/Dockerfile.persistenceagent
+  postCommit: {}
+  source:
+    type: Git
+    git:
+      uri: 'https://github.com/opendatahub-io/ml-pipelines'
```
Comment (anishasthana) [inline manifests/opendatahub/base/buildconfigs/vizualization-server.yaml:31]: This should be red-hat-data-services

File: manifests/opendatahub/base/buildconfigs/vizualization-server.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      ref: master
```
Comment (anishasthana) [inline manifests/opendatahub/base/buildconfigs/vizualization-server.yaml:32]: When we move this over into the release branch we will need to update the branch ref too

**Review by anishasthana — CHANGES_REQUESTED** [review]

**Review by anishasthana — APPROVED** [review]

File: manifests/opendatahub/base/deployments/ml-pipeline-visualizationserver.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@ spec:
         application-crd-id: kubeflow-pipelines
     spec:
       containers:
-        - image: visualization-server
+        - image: visualization-server:1.3.1
```
Comment (gmfrasca) [inline manifests/opendatahub/base/deployments/ml-pipeline-visualizationserver.yaml:22]: I think this image is replaced via the kustomization, so specifying a tag here would just be overwritten, right?  I think we should update the `manifests/opendatahub/base/kustomization.yaml` with this image instead

**Pull request merged**